lib/resources/r4/nutrition_order.yaml

```diff
@@ -0,0 +1,85 @@
+---
+  name: NutritionOrder
+  field_name_base_url: https://www.hl7.org/fhir/nutritionorder-definitions.html#NutritionOrder
+  fields:
+  - name: foodPreferenceModifier
+    required: 'No'
+    type: List of CodeableConcept
+    description: Modifier is used to convey order-specific modifiers about the type of food that should be given.
+    binding:
+      description: Information about a patient's food preference.
+      terminology:
+      - display: Diet
+        system: http://hl7.org/fhir/R4/codesystem-encounter-diet.html
+        info_link: http://hl7.org/fhir/R4/valueset-encounter-diet.html
+      - display: Millennium Dietary Modifiers
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/6102
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-4003-dietary-modifier
+      - display: Millennium Beverage Modifiers
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/6103
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-6103-beverage-modifiers
+  - name: oralDiet.type
+    required: 'No'
+    type: List of CodeableConcept
+    description: The kind of diet or dietary restriction such as fiber restricted diet or diabetic diet.
+    binding:
+      description: The kind of diet or dietary restriction.
+      terminology:
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+        info_link: http://hl7.org/fhir/r4/snomedct.html
+  - name: schedule.code
+    required: 'No'
+    type: CodeableConcept
+    description: A code for the timing schedule.
+    binding:
+      description: Code for a known/defined timing pattern.
+      terminology:
+      - display: TimingAbbreviation
+        system: https://hl7.org/fhir/r4/v3/GTSAbbreviation/cs.html
+        info_link: https://www.hl7.org/fhir/valueset-servicerequest-category.html
```

Comment:
Do you really mean 'ServiceRequest' here?

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/390/commits/15c7aa9c1efdce1ba6756094e8fd868f2de922fa#diff-eb8849a3eca8650b1e50f97e246d9f59R60